Examples/microcontroller/STM32F302R6T6TR.json

```diff
@@ -0,0 +1,11407 @@
+        ]
+      },
+      "esd": true
+    },
+    {
+      "terminalIdentifier": "60",
+      "name": "BOOT0",
+      "numberOfSupportedFunctions": 1,
+      "functionProperties": [
+        {
+          "perFunctionName": "BOOT0"
+        }
+      ],
+      "vih": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "0.2VDD",
+            "relativeValueModifier": 0.95,
+            "relativeValueOperator": "add"
+          }
+        ]
+      },
+      "vil": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "0.3VDD",
+            "relativeValueModifier": -0.3,
+            "relativeValueOperator": "add"
+          }
+        ]
+      },
+      "vol": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "maxValue": 0.4,
+            "conditions": [
+              "CMOS port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 0.4,
+            "conditions": [
+              "TTL port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 1.3,
+            "conditions": [
+              "IIO = +20 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "maxValue": 0.4,
+            "conditions": [
+              "IO = +6 mA",
+              "2 V < VDD < 2.7 V"
+            ]
+          }
+        ]
+      },
+      "voh": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -0.4,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "CMOS port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "minValue": 2.4,
+            "conditions": [
+              "TTL port",
+              "IIO = +8 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -1.3,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IIO = +20 mA",
+              "2.7 V < VDD < 3.6 V"
+            ]
+          },
+          {
+            "siUnit": "volt",
+            "relativeValueReference": "VDD",
+            "relativeValueModifier": -0.4,
+            "relativeValueOperator": "add",
+            "conditions": [
+              "IO = +6 mA",
+              "2 V < VDD < 2.7 V"
+            ]
+          }
+        ]
+      },
+      "absVmax": {
+        "values": [
+          {
+            "siUnit": "volt",
+            "maxV
```

Comment:
good point about millimeter.

I think it makes more sense to use 12x12mm because that's the total size the footprint takes up, but it might be worth discussing in the group - I'll file an issue.